action.yml

```diff
@@ -21,10 +22,17 @@ runs:
   using: 'composite'
   steps:
     - uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c  # v5.0.0
+      if: ${{ inputs.python-version != '' }}
       id: setup-python
       with:
         python-version: ${{ inputs.python-version }}
 
+    - uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c  # v5.0.0
```

Comment:
I like this approach, we've seen python version drift since people forget to update CI when they update `.python-version`